README.md

```diff
@@ -1,6 +1,26 @@
 # colcon-in-container
 
-colcon verb extension to build and test inside a container and transfer the results back to the host
+colcon verb extension to build and test inside a fresh and isolated ROS environment and transfer the results back to the host.
+
+With this extension, developers can build ROS packages for any ROS 2 distributions directly from colcon independently of the host. With it, one can validate builds and tests making sure all the dependencies are properly listed in their `package.xml` on any ROS 2 distribution. Validating packages and workspace in an isolated and ephemeral environment is key for distributing and packaging software.
+
+## Quickstart
+
```

Comment:
```suggestion

This being a colcon extension, make sure to have [colcon installed](https://colcon.readthedocs.io/en/released/user/installation.html).

```

Reply:
This shouldn't be necessary since installing from the extension with pip like presented will install it's dependencies hence colcon.
I would rather add the colcon installation link in the installation section rather than the quickstart.
What do you think?